### ISSUE TITLE
Implement refresh tokens authentication

### ISSUE THREAD

**Issue by iamareebjamal** [fossasia/open-event-attendee-android#2230]
https://root-m29xfdiov.now.sh/

**Comment by liveHarshit** [comment]
So the following are needs to implement:
- Logout and Login with server
- Refresh token (When?)
And the remember-me attribute is for what?

**Comment by iamareebjamal** [comment]
No, logout is not required. Read the documentation carefully and some blog on refresh tokens

**Comment by liveHarshit** [comment]
@anhanh11001 Why it is on hold from server-side?

**Comment by iamareebjamal** [comment]
It's not. This is available in dev server, not in production. All client development must be done using development instance of server

**Comment by liveHarshit** [comment]
@anhanh11001 Are you working on it or can I take this issue?

**Comment by anhanh11001** [comment]
@liveHarshit just a little, please take this issue

**Comment by liveHarshit** [comment]
@iamareebjamal I have one doubt. Here we send refresh token to generate new access token if the token is expired. But which HTTP error is thrown by the server when the access or refresh token is expired. I think it is 404 and 401?

**Comment by ranjsa** [comment]
@liveHarshit Can i work on this issue ?

**Comment by anhanh11001** [comment]
@ranjsa there is already a PR for this, please work on another issue or try to test the app to see if you can find any bugs to fix or reasonable new features to implement

**Comment by spacemonkey1101** [comment]
i would like to work on this if assigned

**Comment by liveHarshit** [comment]
@spacemonkey1101 Take one issue at once

**Comment by ForgottenProgramme** [comment]
> API Docs - https://root-m29xfdiov.now.sh/
> 
> By default, users will not be logged out from the site. All needed info on how to implement it is present in Authentication Section of the API doc above. If any doubt, ask me.

@iamareebjamal , When I try to access the link you have provided, I see this: 
![git](https://user-images.githubusercontent.com/65779580/96337491-2d43d300-10a5-11eb-97c7-2713ad1a9434.png)
